LCM/dsc/engine/ModuleLoader/ModuleLibrary/ModuleHandler.c

```diff
@@ -185,6 +183,7 @@ MI_Result ModuleManager_LoadInstanceDocument (
     *documentInstance = NULL;
 
     /*Load the manager*/
+    (MI_T("%T:%d in %T ~ LoadModuleManager\n"), __FILE__, __LINE__, __FUNCTION__);
```

Comment:
>(MI_T("%T:%d in %T ~ LoadModuleManager\n"), __FILE__, __LINE__, __FUNCTION__) [](start = 4, length = 77)

please remove this line

Reply:
:joy: 

---
In reply to: [263601054](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/541#discussion_r263601054) [](ancestors = 263601054)